tests/e2e/e2e_common.go

```diff
@@ -42,7 +42,7 @@ const (
 	crdtriggercsifullsyncsName                 = "csifullsync"
 	crdGroup                                   = "cns.vmware.com"
 	crdVersion                                 = "v1alpha1"
-	csiSystemNamespace                         = "vmware-system-csi"
+	csiSystemNamespace                         = "kube-system"
```

Comment:
you can pass CSI_NAMESPACE param via env instead right
If some scripts/tests are not working with the ns from env please raise bugs for them